modules/hub/pages/hub-xmlrpc-api-intro.adoc

```diff
@@ -0,0 +1,6 @@
+[[hub-xmlrpc-api]]
+= Hub XMLRPC API
+
+
+The Hub XMLRPC API helps to operate SUSE Manager or Uyuni infrastructures with one Server, called a Hub, managing several Servers.
+Servers are registered and can be managed from the Hub just like normal SUSE Manager or Uyuni clients. In addition to that, the Hub XMLRPC API provides a more convenient way to access all Servers' XMLRPC APIs from one only endpoint in the Hub.
```

Comment:
"In addition to that" seems out of place here - this is the introduction to the Hub XMLRPC API, and only that should be described.